src/reference/asciidoc/jdbc.adoc

```diff
@@ -632,6 +632,14 @@ Such connection pools do normally expect that issued connections are closed with
 For this need of an exclusive connection, it is also recommended that a JVM only runs a single `PostgresChannelMessageTableSubscriber` which can be used to register any number of subscriptions.
 ====
 
+==== Transaction support
+Specifying a `PlatformTransactionManager` on `PostgresSubscribableChannel` will notify subscribers in a transaction.
+An exception in a subscriber will cause the transaction to be rolled back and the message to be put back in the message store.
+Transactional support is not active by default.
+
+==== Retries
+A retry policy can be controlled by providing a `RetryTemplate` to `PostgresSubscribableChannel`. By default, no retries are performed.
```

Comment:
One sentence per line, please: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
done

Reply:
FYI: I prefer to not have these kind of comments in the PR.
If you agree with me, just push the change.
Otherwise feel free to discuss.
In the end this `done` looks like a spam in my e-mail box 😄